### ISSUE TITLE
authority of synonym

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#1977]
[Originally posted on GoogleCode (id 2134) on 2014-02-20]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/8575

<b>What data are incorrect or missing?</b>
Thalictrum polygamum Muhlenberg

<b>What data are you expecting instead?</b>
Thalictrum polygamum Muhlenberg ex Sprengel

<b>If applicable, please provide an authoritative source.</b>
FNA, MBG - however, I see no publication date associated with these entries
IPNI has T. polygamum Muhl. as a nom. nud. 
FNA mentions (in T. pubescens) that T. polygamum is an invalid name.

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2014-02-25 14:15Z]

This needs to be checked: IK often neglected ex authorship in the past and this name appears not to have been verified - there is no GCI entry I could see - TRO mentions 2 entries for IPNI (one with ex) but I could not find it now

I am adding ex for the nonce

nom. inval.: neither TROPICOS or IPNI mention the validity of the name

Therefore I will wait on the nom. inval. but keep it in mind.
Luc

**Comment by brouille** [comment]
Now IPNI has it as TROPICOS and no nom. inval.
VASCAN OK
